src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -397,7 +399,31 @@ private OutputView loopView(
             .map(c -> context.getNamer().localVarName(Name.anyLower(c)))
             .toArray(Object[]::new);
     String formattedComment = String.format(comment, args);
-    ImmutableList<String> lines = ImmutableList.copyOf(formattedComment.split("\\n", -1));
+    // We need to call split with a negative limit because user
+    // might explicitly specify empty comment lines at the end of
+    // the comment block.
+    // But we need to explicitly trim the last element because we don't
+    // want to include the blankline introduced by yaml syntax
+    //
+    // For example:
+    // If yaml looks like:
+    // ```
+    // comment: |
+    //   I am line1
+    //   I am line2
+    // ```
+    // Parsed config would be: `I am line1\nI am line2\n`.
+    //
+    // And splittedComments would be: ["I am line1", "I am line2", ""]
+    //
+    // But we don't want the last empty string.
+    String[] splittedComments = formattedComment.split("\\n", -1);
```

Comment:
s/`splittedComments`/`splitComments`/g 
("split" is the past participle of "to split")

Reply:
And I was hesitating between one 't' and two 't's...